### PR TITLE
fix: hijack time started timestamp fix

### LINKED ISSUE #123: Erroneous timestamp display
**Describe the bug**
The timestamps displayed in the hijack page seem to be inconsistent regarding timezones. This leads to the wrong
assumption that a hijack lasts longer than it did, or that ARTEMIS took more time to detect sth than it really did, and so on.

**Affected Component(s)**
- [ ] Back-End (Node.js, Next.js)
- [X] Front-End (React.js)
- [ ] Theming (CSS, React theme)
- [ ] Build System 
- [ ] Docs

**To Reproduce**
Real hijack, see screenshots and corresponding DB timestamps (which are correct in UTC); the displayed timestamps should be correct in the local GR timezone, but are not consistent.

**Expected behavior**

**Screenshots**
```
vkotronis@skeleton:~/artemis$ docker-compose exec postgres bash
bash-4.4# psql -U artemis_user -d artemis_db
psql (10.5)
Type "help" for help.
artemis_db=# SELECT time_started FROM hijacks WHERE key='dc84c42b7125fcf830c2cf19003f813a';                                                                                                                                                       time_started        
----------------------------
 2021-05-10 11:00:19.043071
(1 row)
artemis_db=# SELECT time_detected FROM hijacks WHERE key='dc84c42b7125fcf830c2cf19003f813a';
       time_detected        
----------------------------
 2021-05-10 12:21:20.873987
(1 row)
artemis_db=# SELECT time_ended FROM hijacks WHERE key='dc84c42b7125fcf830c2cf19003f813a';
     time_ended      
---------------------
 2021-05-10 11:02:12
(1 row)
```

Screenshot 2021-09-03 at 10-47-44 ARTEMIS - Hijack

**System (please complete the following information):**

**Additional context**
Discuss with @vkotronis

### PR DESCRIPTION
**Pull request by CuriouzK0d3r**
Resolves #123 
### Description of PR

What component(s) does this PR affect?

  - [ ] Back-End (Node.js, Next.js)
  - [x] Front-End (React.js)
  - [ ] Theming (CSS, React theme)
  - [ ] Build System
  - [ ] Docs
  
  
  
  Does the PR require changes on other components? If yes, please mark the components:
  
  - [ ] Back-End (Node.js, Next.js)
  - [ ] Front-End (React.js)
  - [ ] Theming (CSS, React theme)
  - [ ] Build System
  - [ ] Docs
  
  
  
### Related Issue
  
  
  
  
  
  
  Resolves #
  
### Type
  
  - [ ] Bug fix (non-breaking change which fixes an issue)
  - [ ] New feature (non-breaking change which adds functionality)
  - [ ] Breaking change (fix or feature that would cause existing functionality to change)
  - [ ] Docs update
  - [ ] None of the above
  
  
  
### Checklist:
  
  - [ ] I have read the **[contributing guide](https://github.com/FORTH-ICS-INSPIRE/artemis/blob/master/CONTRIBUTING.md)** and my code conforms to the guidelines.
  - [ ] This change requires a change in the documentation.
  - [ ] I have updated the documentation accordingly.
  >)`>>>)>

### CODE REVIEW

File: apps/artemis-web/utils/parsers.tsx [hunk 1]
```diff
@@ -312,7 +312,7 @@ function extractHijackInfoLeft(
 function extractHijackInfoRight(hijack) {
   const hijackInfo = {
     'Time Started': [
-      formatDate(new Date(hijack.time_started), 2),
+      formatDate(new Date(hijack.time_started), Math.abs(new Date().getTimezoneOffset() / 60)),
```
Comment (vkotronis) [inline apps/artemis-web/utils/parsers.tsx:315]: please also do the same for mitigation started (not sure why this is not implemented there), and also check again the 'Never' alternative logic in timestamps. In some of them we fall back to 'Never', in others no. All fixes pertain to the `parsers.tsx`. Thanks @CuriouzK0d3r !

Comment (CuriouzK0d3r) [reply apps/artemis-web/utils/parsers.tsx:315]: Nice catch! fixed!

**Review by vkotronis — APPROVED** [review]
Did not have time to test in prod but I assume you have checked it with your local instance. The code fix seems straightforward. Approving.

**Comment by vkotronis** [comment]
this can be merged right?

**Comment by CuriouzK0d3r** [comment]
> this can be merged right?

Yeah, think so :)

**Pull request merged**